packages/palette/src/elements/DataVis/utils/maxInArray.ts

```diff
@@ -0,0 +1,14 @@
+/**
+ * Given an array of numbers or objects returns the max value
+ * @param values array of numbers or objects
+ * @param accessor (optional) when passing an array of objects this function
+ * specifies which attribute of object to be used for comparison. default is a unity function
+ */
+export const maxInArray = (
+  values: any[],
+  accessor: (item: any) => number = item => item
+): number =>
+  values.reduce((currentMax: number, item: any): number => {
+    const value: number = accessor(item)
+    return value > currentMax ? value : currentMax
+  }, -Infinity) as number
```

Comment:
Could https://lodash.com/docs/4.17.11#max or https://lodash.com/docs/4.17.11#maxBy be used instead?

Reply:
yes it is exactly the same as `maxBy`. d3 also has similar think but wasn't sure if we want to add dependency vs adding a 4 line function.

Reply:
We're already using lodash, so that's preferred since we don't need to maintain it.